lldb/source/Plugins/LanguageRuntime/Swift/LLDBMemoryReader.cpp

```diff
@@ -329,16 +329,16 @@ LLDBMemoryReader::resolveRemoteAddress(uint64_t address) const {
   Address resolved(file_address, object_file->GetSectionList());
   if (!resolved.IsValid()) {
     LLDB_LOG(log,
-             "[MemoryReader] Could not make a real address out of file "
-             "address {1:x} and object file {}",
+             "[MemoryReader] Could not make a real address out of file address "
+             "{0:x} and object file {}",
```

Comment:
Just tested this, `{}` will crash, change it to `{1}`

Reply:
done

Reply:
The docs appear to indicate the index isn't required

https://github.com/llvm/llvm-project/blob/287ce6b51675aee43870bebfff68bb144d1ab90e/llvm/include/llvm/Support/FormatVariadic.h#L175

Reply:
Weird, maybe it's a bug then? Or maybe if you specify one position you have to specify all of them?

Reply:
I wondered the same

Reply:
https://reviews.llvm.org/D118833